rfq/negotiator.go

```diff
@@ -541,6 +578,119 @@ func (n *Negotiator) HasAssetSellOffer(assetID *asset.ID,
 	return true
 }
 
+// BuyOffer is a struct that represents an asset buy offer. This data structure
+// describes the maximum amount of an asset that this node is willing to
+// purchase.
+//
+// A buy offer is passive (unlike a buy order), meaning that it does not
```

Comment:
Comment needs update for `BuyOffer` and not `SellOffer`.

Reply:
I don't see the mistake here. This bit is correct:
```
it is used by the node to selectively accept or reject incoming asset sell quote requests
```

The counterparty is requesting a quote to sell their asset.

Reply:
Ah, ok - I was comparing this to the godoc for `SellOffer` and wasn't sure if this was correct.